src/main/java/org/springframework/samples/petclinic/rest/VisitRestController.java

```diff
@@ -78,12 +78,13 @@
 	public ResponseEntity<Visit> addVisit(@RequestBody @Valid Visit visit, BindingResult bindingResult, UriComponentsBuilder ucBuilder){
```

Comment:
**ADV** If we decide to use an @Valid - then we need to add more annotations to the Visit entity class/ Such as @NotBlank(message = "Description is mandatory") description